webapp/src/components/CalendarPage.tsx

```diff
@@ -1,7 +1,51 @@
+import { Container } from '@mui/material';
 import React from 'react';
+import useApiData from '../helpers/useApiData';
+
+import { ProgramWithActivities } from '../types/api';
+import Program from './Program';
 
 const CalendarPage = () => {
-  return <div>Calendar Page</div>;
+  const {
+    data: programs,
+    error,
+    isLoading,
+  } = useApiData<ProgramWithActivities[]>({
+    deps: [],
+    path: '/programs',
+    sendCredentials: true,
+  });
+  if (error) {
+    return <p>There was an error loading the programs.</p>;
+  }
+
```

Comment:
We can tighten up the whitespace on lines 21, 25, and 29.

Reply:
Just pushed a commit that tightens up the whitespace.